examples/namespaced/escape-to-host-si-sib.yaml

```diff
@@ -7,6 +7,9 @@ spec:
     id: escapeToHost
     description: "A attacker can breach container boundaries and can gain access to the host machine"
     action: Block
+    params:
+      exclude_resources: ["app:nginx" ]
+      exclude_caps: ["sys_module" ]
```

Comment:
Looks like this exclude_caps is applicable for the kubeArmor adapter.

We need another exclusion option for each control in the kyverno PSA profile. Another option such as exclude_control needs to be added. This will generate the required fields in podSecurity object of kyerno